### PR TITLE
Add ReadNovelFull

### PR DESCRIPTION
**Pull request by rajarsheechatterjee**

### CODE REVIEW

File: scripts/multisrc/readnovelfull/template.ts [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  name: string;
+  icon: string;
+  site: string;
+  version: string;
+  options?: ReadNovelFullOptions;
+
+  constructor(metadata: ReadNovelFullMetadata) {
+    this.id = metadata.id;
+    this.name = metadata.sourceName;
+    this.icon = `multisrc/readnovelfull/${metadata.id}.png`;
```
Comment (nyagami) [inline scripts/multisrc/readnovelfull/template.ts:30]: this path is incorrect

File: scripts/multisrc/readnovelfull/template.ts [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    this.site = metadata.sourceSite;
+    const versionIncrements = metadata.options?.versionIncrements || 0;
+    this.version = `1.0.${4 + versionIncrements}`;
```
Comment (nyagami) [inline scripts/multisrc/readnovelfull/template.ts:33]:
```suggestion
    this.version = `1.0.${0 + versionIncrements}`;
```

File: scripts/multisrc/readnovelfull/template.ts (hunk 1 above)
Comment (rajarsheechatterjee) [reply scripts/multisrc/readnovelfull/template.ts:30]: Done

File: scripts/multisrc/readnovelfull/template.ts (hunk 2 above)
Comment (rajarsheechatterjee) [reply scripts/multisrc/readnovelfull/template.ts]: Done

**Pull request merged**